src/Interface/CheckInsertInterface.php

```diff
@@ -11,8 +11,10 @@
 
 namespace Manticoresearch\Buddy\Interface;
 
+//@codingStandardsIgnoreStart
```

Comment:
We should remove comments like this due to it's fixed already in the ruleset.xml, and PHPCS will not remove them when they are used only in comment blocks for type hinting